docs/server/source/cloud-deployment-starter-templates/new-azure-template.md

```diff
@@ -0,0 +1,57 @@
+# New Azure Template
+
+If you didn't read the introduction to the [cloud deployment starter templates](index.html), please do that now. The main point is that they're not for deploying a production node; they can be used as a starting point.
+
+One can deploy a BigchainDB node on Azure using the template in the `bigchaindb-on-ubuntu` directory of Microsoft's `azure-quickstart-templates` repository on GitHub:
+
+1. Go to [the `bigchaindb-on-ubuntu` directory in that repository](https://github.com/Azure/azure-quickstart-templates/tree/master/bigchaindb-on-ubuntu).
+
+2. Click the button labelled **Deploy to Azure**.
+
+3. If you're not already logged in to Microsoft Azure, then you'll be prompted to login. If you don't have an account, then you'll have to create one.
+
+4. One you are logged in to the Microsoft Azure Portal, you should be taken to a form titled **BigchainDB**. There are some notes to help with filling in that form at the bottom of this page.
+
+5. Deployment takes a few minutes. You can follow the notifications by clicking the bell icon at the top of the screen. When done, you should see a notification saying "Deployment to resource group '[your resource group name]' was successful." The install script (`init.sh`) installed RethinkDB, configured it using the default RethinkDB config file, and ran it. It also used `pip` to install [the latest `bigchaindb` from PyPI](https://pypi.python.org/pypi/BigchainDB).
+
+6. Find out the public IP address of the virtual machine in the Azure Portal. Example: `40.69.87.250`
```

Comment:
Sadly, the virtual machine's IP address is not at all obvious. It takes some digging to find it. (There are many services launched and the virtual machine is only one of them. In fact, another one of the services is the static IP address and one could look there instead...)